ledfx/effects/texter2d.py

```diff
@@ -275,19 +281,31 @@ def side_scroll_init(self):
             sentence_width -= self.sentence.space_block.w_width
             offset = -sentence_width / 2
 
+        direction = 1 if not self.rtl_direction else -1
```

Comment:
I think direction should be self.direction configured in the config code, it doesn't need to be calculated out each frame

Reply:
Noting I need to look from PC, doing this from phone risks me missing class / scope implications